### PR TITLE
Upgrade formatter-maven-plugin to 2.17.1

### PR DESCRIPTION
**Pull request by jamesnetherton**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: poms/build-parent/pom.xml [hunk 1]
```diff
@@ -97,6 +97,7 @@
                     <version>${formatter-maven-plugin.version}</version>
                     <configuration>
                         <configFile>${maven.multiModuleProjectDirectory}/tooling/eclipse-formatter-config.xml</configFile>
+                        <lineEnding>KEEP</lineEnding>
```
Comment (ppalaga) [inline poms/build-parent/pom.xml:100]: What's the meaning of this change? Would the plugin keep Windows EOLs if end users enter them?

Comment (ppalaga) [reply poms/build-parent/pom.xml:100]: Oh really? Keep is certainly better than user's platform. Is there no way to force Unix EOLs?

Comment (jamesnetherton) [reply poms/build-parent/pom.xml:100]: >  Is there no way to force Unix EOLs?

Yes, I guess it'd make more sense to use `<lineEnding>LF</lindending>`.

Comment (jamesnetherton) [reply poms/build-parent/pom.xml]: Latest commit forces Unix EOLs.

**Pull request merged**